### ISSUE TITLE
Build requirements?

### ISSUE THREAD

**Issue by Ark74** [husisusi/officeonlin-install.sh#3]
Hi!
This is not really an issue or bug, more like a request.
I've read some places that this script require some decent resources and time to be able to build.
Could you suggest some minimum specs to address before any attempt?
I've read, 4 GB ram, somewhat 16GB on disk or so, but not really sure the real numbers.

Thanks.

**Comment by husisusi** [comment]
Hi Ark74,
I tested it with 4 GB RAM and 20 GB disk and it works.
Compiling LibreOffice require really CPU resource and take a long time. Everything else goes pretty fast. 
More resource means faster Compiling and that's it. I do not believe that you will get some errors due to lack of resources. Nevertheless, let us say 4 GB RAM and 20 GB disk.

More interesting is lool resources require by using LibreOffice Online . I did some tests and seemingly 4 GB is enough:
https://help.nextcloud.com/uploads/default/original/2X/8/8c4167e8ec5d23409730368f541af207a379866d.png

Good Luck!
